src/module/apikeys/index.ts

```diff
@@ -1,29 +1,29 @@
-export const UserID = "user.venturemark.co/id";
-export const VentureID = "venture.venturemark.co/id";
-export const TimelineID = "timeline.venturemark.co/id";
-export const UpdateID = "update.venturemark.co/id";
-export const MessageID = "message.venturemark.co/id";
-export const RoleID = "role.venturemark.co/id";
-export const InviteID = "invite.venturemark.co/id";
-export const SubjectID = "subject.venturemark.co/id";
+export const UserID = "user.breadcrumb.so/id";
```

Comment:
Please revert the renaming of these constants. They look like URLs, but they're really just "namespaced" constants like what Kubernetes uses to avoid naming collisions. This would require migrations and apiserver changes so let's keep them as venturemark.co for now.

Reply:
ok sure